From e4c5b8f9c0532f43eb4bacb5959becab015ca082 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 23 May 2011 14:54:03 +0200
Subject: [PATCH] ui-utils-contacts: check returned item from
 elm_list_item_append

* gcc-4.6 doesn't like unused variables which are never read

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/util/ui-utils-contacts.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/util/ui-utils-contacts.c b/src/util/ui-utils-contacts.c
index 2614ec5..961190e 100644
--- a/src/util/ui-utils-contacts.c
+++ b/src/util/ui-utils-contacts.c
@@ -139,6 +139,9 @@ _add_number_to_list_real(struct _number_select_pack *pack,
 	elm_icon_scale_set(ico, 1, 1);
 	elm_icon_file_set(ico, phoneui_theme, "icon/phone");
 	it = elm_list_item_append(pack->list, entry->number, ico, NULL, NULL, entry);
+	if (!it) {
+		g_warning("Adding number to list failed!!!");
+	}
 }
 
 static void
-- 
1.7.5.rc3

